lib/reply.js

```diff
@@ -163,7 +165,14 @@ Reply.prototype.send = function (payload) {
   const hasContentType = contentType !== undefined
 
   if (payload !== null) {
-    if (typeof payload.pipe === 'function') {
+    if (
+      // node:stream
+      typeof payload.pipe === 'function' ||
+      // node:stream/web
+      typeof payload.getReader === 'function' ||
+      // Response
+      Object.prototype.toString.call(payload) === '[object Response]'
```

Comment:
Why not:

```suggestion
      payload instanceof Response
```

Is it to support node 16?

Reply:
> Is it to support node 16?

No, I want to support `polyfill` or non-native implementation of `Response`.
`instanceof` will be restricted to only the `node` provided one, not event `undici`.